index.html

```diff
@@ -184,5 +264,102 @@ <h4 class="top-banner__subtitle">
         </div>
       </div>
     </section>
+    <section class="why-us">
+      <div class="why-us__container">
```

Comment:
I suggest you to use the already existing class "container". No need to create a new one. You could use your "container" in any future section